examples/webview2.rs

```diff
@@ -0,0 +1,94 @@
+use com::interfaces::IUnknown;
+use com::sys::{HRESULT, S_OK};
+use std::sync::{Arc, Mutex};
+
+com::interfaces! {
+    #[uuid("DA66D884-6DA8-410E-9630-8C48F8B3A40E")]
+    pub unsafe interface ICoreWebView2Environment : IUnknown {}
+
+    #[uuid("8B4F98CE-DB0D-4E71-85FD-C4C4EF1F2630")]
+    pub unsafe interface ICoreWebView2CreateCoreWebView2EnvironmentCompletedHandler : IUnknown {
+        unsafe fn invoke(
+            &self,
+            result: HRESULT,
+            created_environment: ICoreWebView2Environment
+        ) -> HRESULT;
+    }
+}
+
+com::class! {
+    pub class CreateCoreWebView2EnvironmentCompletedHandler : ICoreWebView2CreateCoreWebView2EnvironmentCompletedHandler {
+        destination: Arc<Mutex<Option<ICoreWebView2Environment>>>,
+    }
+
+    impl ICoreWebView2CreateCoreWebView2EnvironmentCompletedHandler for CreateCoreWebView2EnvironmentCompletedHandler {
+        fn invoke(
+            &self,
+            result: HRESULT,
+            created_environment: ICoreWebView2Environment // error: mismatched types (expects type on next line)
+            // created_environment: std::ptr::NonNull<std::ptr::NonNull<ICoreWebView2EnvironmentVTable>> // works
```

Comment:
If this is going to become a persistent example and/or potential regression test then I'm not sure that these comments are helpful as is.  For a regression test they might be helpful if they explicitly labeled which line indicates that things are (now) working as expected and which line indicates a regression, but that's kind of the opposite of what they currently say.  Maybe line 28 should say something like "expected to work" and line 29 could say something like "if this is needed instead then ABI translation is probably broken".